scripts/secnetperf-helpers.psm1

```diff
@@ -511,7 +529,7 @@ function Invoke-Secnetperf {
 
     # Start the server running.
     "> secnetperf $serverArgs" | Add-Content $serverOut
-    $job = Start-RemoteServer $Session "$RemoteDir/$SecNetPerfPath $serverArgs"
+    $job = Start-RemoteServer $Session "$RemoteDir/$SecNetPerfPath $serverArgs" (!$IsWindows -and $io -eq "xdp")
```

Comment:
1. Since this condition never changes, let's make a `$sudo` variable above for this, equal to "" or "sudo MSQUIC_ENABLE_XDP=1".
2. Instead of changing the functions above, just inject `$sudo` here and below.
```suggestion
    $job = Start-RemoteServer $Session "$sudo $RemoteDir/$SecNetPerfPath $serverArgs"
```
Example change below in other comment.